PiwikTracker/Context.swift

```diff
@@ -0,0 +1,115 @@
+import UIKit
+
+public class Context {
+    
+    public class Application {
+        // TODO: implement me
+    }
+    
+    public class Device {
```

Comment:
Consider asking access control (internal) to the top level type to be consistent with the other PR.
I think this code style guide is pretty good.
https://github.com/raywenderlich/swift-style-guide
https://github.com/github/swift-style-guide